gooddata-sdk/gooddata_sdk/catalog/user/entity_model/user.py

```diff
@@ -53,27 +53,62 @@ def init(
 
     @property
     def user_groups(self) -> List[CatalogUserGroup]:
+        """Get method for the cached user_groups
+
+        Returns:
+            List[CatalogUserGroup]:
+                List of User Groups
+        """
         if self.relationships and self.relationships.user_groups:
             return self.relationships.user_groups.data
         return []
 
     def add_user_group(self, user_group: CatalogUserGroup) -> None:
+        """Adds a User Group to the relationships variable.
+
+        Args:
+            user_group (CatalogUserGroup):
+                User Group to be added.
+        Returns:
+            None
+        """
         if not self.relationships:
             self.relationships = CatalogUserRelationships(user_groups=CatalogUserGroupsData(data=[user_group]))
         else:
             self.relationships.add_user_groups([user_group])
 
     def add_user_groups(self, user_groups: list[CatalogUserGroup]) -> None:
+        """Adds multiple User Groups to relationships variable.
```

Comment:
I would rephrase it as:

Adds multiple User Groups to the relationship with the user.